tests/test_regression.py

```diff
@@ -1,100 +1,333 @@
 #! /usr/bin/env python
 
+from nose.tools import assert_equal, assert_true
 import os
-import json
-import hashlib
-import urllib
+from unittest import TestCase
+import tables
+import numpy as np
+from numpy.testing import assert_almost_equal 
```

Comment:
They seem to do the same thing and `assert_array_almost_equal()` doesn't conflict with a name in nose.